pkgs/by-name/wo/wordfence-cli/package.nix

```
@@ -0,0 +1,91 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  pkg-config,
+  pcre,
+  hyperscan,
```

```suggestion
  vectorscan,
```

See https://github.com/wordfence/wordfence-cli/blob/main/docs/malware-scan/Vectorscan.md

Opens up more platforms